src/loader/loader.c

```diff
@@ -76,7 +72,15 @@ PG_MODULE_MAGIC;
 #endif
 
 #define GUC_DISABLE_LOAD_NAME "timescaledb.disable_load"
-
+#if PG96
+#ifdef WIN32
+#define CalledInParallelWorker() false
+#else
+#define CalledInParallelWorker() IsParallelWorker()
+#endif
```

Comment:
please add comment /* WIN32 */ to make it easier to read. what the endif is for since we don't indent